draft-ietf-quic-http.md

```diff
@@ -472,127 +475,125 @@ or complete response but does not abort reading, clients SHOULD continue sending
-    includes the value '*' for the ":path" pseudo-header field.
+    includes the value '*' for the ":path" pseudo-field.
 
-  : This pseudo-header field MUST NOT be empty for "http" or "https" URIs;
+  : This pseudo-field MUST NOT be empty for "http" or "https" URIs;
     "http" or "https" URIs that do not contain a path component MUST include a
     value of '/'.  The exception to this rule is an OPTIONS request for an
     "http" or "https" URI that does not include a path component; these MUST
-    include a ":path" pseudo-header field with a value of '*' (see Section 5.3.4
-    of [RFC7230]).
+    include a ":path" pseudo-field with a value of '*' (see Section 3.2.4
+    of {{?HTTP11}}).
 
 All HTTP/3 requests MUST include exactly one value for the ":method", ":scheme",
-and ":path" pseudo-header fields, unless it is a CONNECT request ({{connect}}).
-An HTTP request that omits mandatory pseudo-header fields or contains invalid
-values for those fields is malformed ({{malformed}}).
+and ":path" pseudo-fields, unless it is a CONNECT request ({{connect}}).
+An HTTP request that omits mandatory pseudo-fields or contains invalid
+values for those pseudo-fields is malformed ({{malformed}}).
 
 HTTP/3 does not define a way to carry the version identifier that is included in
 the HTTP/1.1 request line.
 
-For responses, a single ":status" pseudo-header field is defined that carries
-the HTTP status code field (see Section 6 of [RFC7231]).  This pseudo-header
-field MUST be included in all responses; otherwise, the response is malformed
+For responses, a single ":status" pseudo-field is defined that carries the HTTP
+status code (see Section 9 of {{!SEMANTICS}}).  This pseudo-field
+MUST be included in all responses; otherwise, the response is malformed
 ({{malformed}}).
 
 HTTP/3 does not define a way to carry the version or reason phrase that is
 included in an HTTP/1.1 status line.
 
-#### Header Compression
+#### Field Compression
 
-HTTP/3 uses QPACK header compression as described in [QPACK], a variation of
-HPACK which allows the flexibility to avoid header-compression-induced
-head-of-line blocking.  See that document for additional details.
+HTTP/3 uses QPACK field compression as described in [QPACK], a variation of
+HPACK which allows the flexibility to avoid compression-induced head-of-line
+blocking.  See that document for additional details.
 
-To allow for better compression efficiency, the cookie header field {{!RFC6265}}
-MAY be split into separate header fields, each with one or more cookie-pairs,
-before compression. If a decompressed header list contains multiple cookie
+To allow for better compression efficiency, the "Cookie" field {{!RFC6265}}
+MAY be split into separate field lines, each with one or more cookie-pairs,
+before compression. If a decompressed field list contains multiple cookie
 header fields, these MUST be concatenated into a single octet string using the
```

Comment:
```suggestion
fields, these MUST be concatenated into a single octet string using the
```

Reply:
Should be "field lines," actually.  Good catch!